src/thriftpyi/stubs.py

```diff
@@ -30,6 +30,83 @@ def build_init(imports: Iterable[str]) -> ast.Module:
     )
 
 
+def build_typedefs() -> ast.Module:
+    types_map = {
+        "Binary": ("bytes", "BINARY"),
+        "Bool": ("bool", "BOOL"),
+        "Byte": ("int", "BYTE"),
+        "Double": ("float", "DOUBLE"),
+        "I16": ("int", "I16"),
+        "I32": ("int", "I32"),
+        "I64": ("int", "I64"),
+        "String": ("str", "STRING"),
+    }
+
+    body: list[ast.stmt] = [
+        ast.ImportFrom(
+            module="dataclasses",
+            names=[ast.alias(name="dataclass")],
+            level=0,
+        ),
+        ast.ImportFrom(
+            module="typing",
+            names=[ast.alias(name="Annotated")],
+            level=0,
+        ),
+        ast.ClassDef(
+            name="Metadata",
+            bases=[],
+            keywords=[
```

Comment:
I think you want to move this to the `dataclass` below

Reply:
fixed :)